index.bs

```diff
@@ -651,8 +651,16 @@ enum ReadableStreamType { "bytes" };
   <p>For an example of how to set up a readable byte stream, including using the different
   controller interface, see [[#example-rbs-push]].
 
-  <p>Setting any value other than "{{ReadableStreamType/bytes}}" or undefined will cause the
-  {{ReadableStream()}} constructor to throw an exception.
+  <p>Can be set to "<dfn enum-value for="ReadableStreamType">owning</dfn>" to signal that the
+  constructed {{ReadableStream}} will own chunks (via transfer or serialization) before enqueuing them.
+  This ensures that enqueued chunks are not mutable by the source.
+  Transferred or serialized chunks may have <dfn>dispose steps</dfn> which are executed if
```

Comment:
```suggestion
  Transferred or serialized chunks may have <dfn export>dispose steps</dfn> which are executed if
```

Reply:
BTW, chunks should already have dispose steps before transfer/serialization, so this phrase looks a bit off. Can it be:

>Chunks may have dispose steps which are executed when they are owned by the stream via enqueuing and then dequeued without ...

Reply:
Will update the sentence and add export.